facia-tool/public/js/utils/article-path.js

```diff
@@ -0,0 +1,14 @@
+import urlHost from './url-host';
+import urlAbsPath from './url-abs-path';
+
+export default function(url) {
+
+    var host = urlHost(url);
+
+    if (host === 'viewer.gutools.co.uk') {
```

Comment:
can you move `viewer.gutools.co.uk` to `facia-tool/public/js/constants/defaults.js` please